charts/consul/templates/crd-ingressgateways.yaml

```diff
@@ -78,6 +78,40 @@ spec:
                       while waiting for a connection to be established.
                     format: int32
                     type: integer
+                  passiveHealthCheck:
+                    description: PassiveHealthCheck configuration determines how upstream
+                      proxy instances will be monitored for removal from the load
+                      balancing pool.
+                    properties:
+                      baseEjectionTime:
+                        description: The base time that a host is ejected for. The
+                          real time is equal to the base time multiplied by the number
+                          of times the host has been ejected and is capped by max_ejection_time
+                          (Default 300s). Defaults to 30000ms or 30s.
```

Comment:
I would just say `30s` instead of listing it in two different units. Especially since the `max_ejection_time` is given in seconds. Also, seconds is just a more useful level of accuracy at this level.

Reply:
done